packages/main/src/Button.hbs

```diff
@@ -28,6 +28,7 @@
 			accessible-role="{{iconRole}}"
 			part="icon"
 			?show-tooltip={{showIconTooltip}}
+			@click="{{_onIconClick}}"
```

Comment:
The Icon click is a custom event - in this case listening to "ui5-click" is recommended (as it will be fired always, even in "noConflict" mode)